### ISSUE TITLE
Formatting fails with elm-format 0.8.1-rc1

### ISSUE THREAD

**Issue by sporto** [Krzysztof-Cieslak/vscode-elm#270]
I installed `elm-format 0.8.1-rc1`.
When trying to format the plugin-in started failing with "Running elm-format failed. Install from ..."

But the real issue is that this new version of elm-format requires a flag for the elm version.
Maybe the plugin could print the actual error from elm-format if possible. Which in the command line is:

```
I couldn't figure out what version of Elm you are using!

You should either run elm-format from the directory containing
your elm.json (for Elm 0.19) or elm-package.json (for Elm 0.18),
or tell me which version of Elm with --elm-version=0.19 or --elm-version=0.18
```

Changing the command to `elm-format  --elm-version=0.19` fixed the issue for me.

**Comment by hakonrossebo** [comment]
#262 should fix this issue when the new release is out.

**Comment by Krzysztof-Cieslak** [comment]
Fixed in `0.22.`
